ts/editor/rich-text-input/RichTextInput.svelte

```diff
@@ -22,7 +21,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         inputHandler: InputHandlerAPI;
         /** The API exposed by the editable component */
         editable: ContentEditableAPI;
-        customStyles: Promise<CustomStyles>;
+        customStyles: Promise<Record<string, any>>;
```

Comment:
Couldn't figure out the right way to type components in Svelte 5.

Reply:
No worries, I would have done the same. :-)